config/template.go

```diff
@@ -48,7 +82,12 @@ func defaultValue(defaultValue, templateValue interface{}) string {
 func NewTemplate(config []byte) (*Template, error) {
 	env := parseEnvironment(os.Environ())
 	tmpl, err := template.New("").Funcs(template.FuncMap{
-		"default": defaultValue,
+		"default":       defaultValue,
+		"split":         split,
+		"join":          join,
+		"replace":       replace,
+		"regexReplace":  regexReplace,
+		"regexpReplace": regexReplace,
```

Comment:
Why the alias? If we were going to have more than one regex option at all I'd rather we match what `consul-template` did with having both a [`regexMatch`](https://github.com/hashicorp/consul-template#regexmatch) and [`regexReplaceAll`](https://github.com/hashicorp/consul-template#regexreplaceall).

If we don't want to implement the `regexMatch` bool (which doesn't seem unreasonable), then having the name be `regexReplaceAll` has the added advantage of giving us room to have a "replace _n_" option down the road.